app/src/main/java/de/danoeh/antennapod/adapter/FeedItemlistDescriptionAdapter.java

```diff
@@ -67,8 +71,14 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
             holder.description.setText(description);
             holder.description.setMaxLines(MAX_LINES_COLLAPSED);
         }
+        holder.title.setMaxLines(2);
+        holder.description.setMaxLines(2);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            holder.title.setTypeface(Typeface.create(null, 700, false));
+            holder.preview.setTypeface(Typeface.create(null, 600, false));
+        }
```

Comment:
Can't this be set in xml? We should try to separate design and functionality. Changing the design from Java mixes these

Reply:
This was done because these are added in XML but are not applying so had to go with this.